.jvmopts

```diff
@@ -1,4 +1,5 @@
 -Dfile.encoding=UTF8
+-Djava.security.egd=file:/dev/./urandom
```

Comment:
@raboof Very interesting article. And, yes, this was a tough work indeed. 4 hours of hard debug session for just a new line on a configuration file. This job is ridicule sometimes...